Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
I uninstalled all of my gems, checked out the previous commit ran bundle then checkout out this commit and ran bundle again to pick up `will_paginate_mongoid` and I get the same resulting diff on `Gemfile.lock`. So I think this diff is ok.

There was some conversation if a gem should check in  `Gemfile.lock` at all http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/ and https://github.com/rails/rails/commit/a437986f433a775c7c370ad3f0273938015699df#commitcomment-731274